messaging/src/main/java/org/axonframework/common/AxonRetriesExhaustedException.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2010-2022. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.common;
+
+import java.io.Serializable;
+
+/**
+ * Exception indicating that an action was retried a maximum times, without a good result.
+ *
+ * @author Gerard Klijs
+ * @since 4.5.13
+ */
+public class AxonRetriesExhaustedException extends AxonException implements Serializable {
```

Comment:
How about we call this the `ProcessRetriesExhaustedException`? :thinking:

Reply:
done